portfolio/src/main/webapp/index.html

```diff
@@ -1,31 +1,64 @@
 <!DOCTYPE html>
 <html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Giovanni's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
 
-<head>
-	<meta charset="UTF-8">
-	<title>Giovanni's Portfolio</title>
-	<link rel="stylesheet" href="style.css">
-	<script src="script.js"></script>
-</head>
-
-<body>
-	<div id="content">
-		<h1>Giovanni's Portfolio</h1>
-		<p>Hello, I am a Computer Engieering Student, at the University of Florida. </p>
-		<p>Click here to get a random message:</p>
-		<button onclick="randomFact()">click for a random Gio fact.</button>
-		<p>Click <a href="images/images.html">here</a> to see more pictures of me!</p>
-		<p>Click <a href="blogs/blogs.html">here</a> to see what people have written about me!</p>
-		<p>Click <a href="Projects/Projects.html">here</a> to see my projects(🚧Under Construction!🚧)</p>
-
-		<p> <a href="Resume.pdf">RESUME</a>.</p>
-		<p> <a href="https://www.linkedin.com/in/giovanni-ferioli">My LinkedIn</a> </p>
-		<div id="fact-container"></div>
-		<button onclick="getName()">Use fetch()</button>
-		<div id="greeting-container"></div>
-        <div id="list-container"></div>
-	</div>
-
-</body>
+  <body>
+    <div id="content">
+      <h1>Giovanni's Portfolio</h1>
+
+      <p>Hello, I am a Computer Engineering Student, at the University of Florida. </p>
+
+      <p>Click here to get a random message:</p>
+
+      <button onclick="randomFact()">click for a random Gio fact.</button>
+
+      <p>Click <a href="images/images.html">here</a> to see more pictures of me!</p>
+      <p>Click <a href="blogs/blogs.html">here</a> to see what people have written about me!</p>
+      <p>Click <a href="Projects/Projects.html">here</a> to see my projects(🚧Under Construction!🚧)</p>
+
+      <p> <a href="Resume.pdf">RESUME</a>.</p>
+      <p> <a href="https://www.linkedin.com/in/giovanni-ferioli">My LinkedIn</a> </p>
+
+      <div id="fact-container"></div>
+
+      <button onclick="getName()">Use fetch()</button>
+      <div id="greeting-container"></div>
+      <div id="list-container"></div>
```

Comment:
I would add either a `<ul>` or an `<ol>` element here and use that instead to add child `<li>` elements.

Reply:
this is still applicable FYI